hyperspy/io.py

```diff
@@ -124,6 +124,10 @@ def load(filenames=None,
         mapped file will be created in the given directory,
         otherwise the default directory is used.
 
+    load_to_memory: bool
```

Comment:
All readers could support `load_to_memory=False` by just reading the array as a `numpy.memmap`. Currently this is the default for `mrc` files.